crates/common/src/lib.rs

```diff
@@ -298,12 +298,18 @@ impl TransactionVersion {
     pub const ONE: Self = Self(H256(hex_literal::hex!(
         "0000000000000000000000000000000000000000000000000000000000000001"
     )));
+    pub const TWO: Self = Self(H256(hex_literal::hex!(
+        "0000000000000000000000000000000000000000000000000000000000000002"
+    )));
     pub const ZERO_WITH_QUERY_VERSION: Self = Self(H256(hex_literal::hex!(
         "0000000000000000000000000000000100000000000000000000000000000000"
     )));
     pub const ONE_WITH_QUERY_VERSION: Self = Self(H256(hex_literal::hex!(
         "0000000000000000000000000000000100000000000000000000000000000001"
     )));
+    pub const TWO_WITH_QUERY_VERSION: Self = Self(H256(hex_literal::hex!(
+        "0000000000000000000000000000000100000000000000000000000000000002"
+    )));
```

Comment:
For query version we can add a `const fn` to set the version bit:
```suggestion
    const fn with_query_flag(mut self) -> Self {
        self.0.0[16] = 0b1000_0000;
    }

    pub const TWO_WITH_QUERY_VERSION: Self = Self::TWO.with_query_flag();
```